xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/job/DeleteJob.java

```diff
@@ -147,4 +147,14 @@ private void maybeDelete(DocumentReference documentReference)
             this.logger.debug("[{}] has been successfully deleted.", documentReference);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     * 
+     * @see org.xwiki.refactoring.internal.job.AbstractEntityJob#getCommonParent()
+     */
+    public EntityReference getCommonParent()
+    {
+        return getCommonParent(this.request.getEntityReferences());
+    }
```

Comment:
Why is this overwrite needed? Just to make it public? Then why not return ``super.getCommonParent()``? BTW, I'm not sure the reason you made this public is good. See below.